bssl-compat/prefixer/prefixer.cpp

```diff
@@ -112,7 +112,7 @@ class Function {
 
     std::string getHeader(const clang::SourceManager &srcmgr) const {
       clang::SourceLocation sloc = getFileLoc(srcmgr, m_node->getLocation());
-      const clang::FileEntry *declfile = srcmgr.getFileEntryForID(srcmgr.getFileID(sloc));
+      const clang::OptionalFileEntryRef declfile = srcmgr.getFileEntryRefForID(srcmgr.getFileID(sloc));
```

Comment:
Strictly speaking, this change and the one below are only required in clang 19. In clang 18 those API's are deprecated but still valid. I realized this when trying rockylinux and the version of clang that comes with it (19).

I think it's safe to merge them now instead of waiting until the next clang bump and wasting time debugging what's broken. I've already spent that time :D

Reply:
Have you happened to check if clang 17 is also OK with this notation?
Until we move build to ubi, we need to be compatible with it.